app/controller.js

```diff
@@ -5,8 +5,22 @@ novapostaApp.controller('novapostaController', function novapostaController($sco
         $scope.Description= $routeParams.Description;
 
     });
+    $scope.onMap = function (city){
+
+        novapostaService.onMap(function (data) {
+
+        }, city)
+    }
+
+
+
 });
 
+
+
+
+
+
 novapostaApp.controller('cityController', function cityController($scope, cityService, $routeParams) {
```

Comment:
You should wrap params into array and remove function name. It can be anonymous.